src/components/LoadMoreButton/LoadMoreButton.tsx

```diff
@@ -2,8 +2,22 @@ import React from "react";
 import "./LoadMoreButton.css";
 
 interface LoadMoreButtonProps {
-  onClick: () => void;
+  onClick?: () => void;
+  isEnabled?: boolean;
 }
-export const LoadMoreButton = ({ onClick }: LoadMoreButtonProps) => {
-  return <div className="load-more-button casts-shadow" onClick={onClick}>Load More</div>;
+export const LoadMoreButton = ({
+  onClick,
+  isEnabled = true,
+}: LoadMoreButtonProps) => {
+  return (
+    <button
+      className={`load-more-button ${isEnabled && "button-enabled"} `}
```

Comment:
In case of isEnabled false, undefined, and null their respective classes will be added.
```suggestion
      className={`load-more-button ${isEnabled ? "button-enabled" : ''} `}
```